axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConnectionManager.java

```diff
@@ -123,6 +145,11 @@ protected AxonServerConnectionManager(Builder builder) {
         this.axonServerConfiguration = builder.axonServerConfiguration;
         this.tagsConfiguration = builder.tagsConfiguration;
         this.scheduler = builder.scheduler;
+        onOutboundInstruction(NODE_NOTIFICATION,
```

Comment:
You're calling a deprecated operation here. Change to handers.register?

Reply:
You are right, I wanted to invoke `public void onOutboundInstruction(PlatformOutboundInstruction.RequestCase requestCase,                                   BiConsumer<PlatformOutboundInstruction, StreamObserver<PlatformInboundInstruction>> handler)`